src/core/MOM_grid.F90

```diff
@@ -112,6 +112,16 @@ module MOM_grid
     IareaCv, &   !< The masked inverse areas of v-grid cells [L-2 ~> m-2].
     areaCv       !< The areas of the v-grid cells [L2 ~> m2].
 
+  real ALLOCABLE_, dimension(NIMEMB_PTR_,NJMEM_) :: &
```

Comment:
These heights should be stored in units of [Z ~> m], not [m], and they should also use an offset (reference height) given by GV%Z_ref.  (The value of depths stored in G%bathyT is usually the value read from a file minus GV%Z_ref, and you would subtract GV%Z_ref from these heights after they are read in.)  By default most cases use Z_to_m = 1.0 and GV%Z_ref = 0.0, but this modest change would bring the new porous barriers code into the coverage of the dimensional consistency tests, and it is something that we have found invaluable in preparing the code to work with wetting and drying.